delete_by_query.py

```diff
@@ -46,13 +60,13 @@ def delete_docs(es_url, index, doc_type, query):
                     "_type": doc_type,
                     "_id": rec['_id']
                 })
-            pprint(bulk_body)
             print('Deleting records')
             helpers.bulk(es, bulk_body)
             print('Delete successful')
 
     except exceptions.NotFoundError:
-        pass
+        print("Elasticsearch error: " + ex.error)
```

Comment:
It is better to use logger (http://docs.python-guide.org/en/latest/writing/logging/), maybe in a separate PR.